### PR TITLE
Hsinyun

### PR DESCRIPTION
**Pull request by hsin-yunn**
1. shop_campaign
2. shop_campaign_shop_order

### CODE REVIEW

File: src/Controllers/ShopCampaignController.php [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+use App\Http\Controllers\Controller;
+use Carbon\Carbon;
+use Wasateam\Laravelapistone\Helpers\ModelHelper;
+
+/**
+ * @group 促銷活動 ShopCampaign
+ *
+ * 促銷活動 API
+ *
+ * 類型 type
```
Comment (davidturtle) [inline src/Controllers/ShopCampaignController.php:14]: 這裡的說明跟migration好像沒有完全對上

File: src/Controllers/ShopCampaignShopOrderController.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Wasateam\Laravelapistone\Helpers\ModelHelper;
+
+/**
+ * @group ShopCampaignShopOrder
+ *
+ * ShopCampaignShopOrder API
+ *
+ * user
```
Comment (davidturtle) [inline src/Controllers/ShopCampaignShopOrderController.php:14]: 沒寫完整

Comment (hsin-yunn) [reply src/Controllers/ShopCampaignShopOrderController.php]: done

File: src/Controllers/ShopCampaignController.php (hunk 1 above)
Comment (hsin-yunn) [reply src/Controllers/ShopCampaignController.php:14]: done

**Pull request merged**